op-chain-ops/cmd/check-l2/main.go

```diff
@@ -36,6 +36,9 @@ var (
 	// errAlreadyInitialized represents a revert from when a contract is already initialized.
 	// This error is used to assert with `eth_call` on contracts that are `Initializable`
 	errAlreadyInitialized = errors.New("Initializable: contract is already initialized")
+	// errInvalidInitializing represents when the initializing value is not set to the expected value.
+	// This is an assertion on `_initializing`. We do not care about the value of `_initialized`.
+	errInvalidInitializing = errors.New("invalid initializing value")
```

Comment:
It would be more ideal to get this error on develop so that all downstream PRs in this bunch could use it but https://github.com/ethereum-optimism/optimism/pull/8933 will allow us to delete `check-l2` in a follow up